examples/tests/main.tf

```diff
@@ -0,0 +1,157 @@
+####################################
+# Test case 1 - Unmanaged OIDC public cluster
```

Comment:
Lets make this example more clear:
1. Change the directory name to meaningful name
2. Add README with more detailed description about what this example does
3. Please remove all commented lines